test/e2e/scheduling/predicates.go

```diff
@@ -95,17 +95,14 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 
 		framework.AllNodesReady(cs, time.Minute)
 
-		// NOTE: Here doesn't get nodeList for supporting a master nodes which can host workload pods.
-		masterNodes, _, err = e2enode.GetMasterAndWorkerNodes(cs)
```

Comment:
Can we remove e2enode.GetMasterAndWorkerNodes(), or mark it as deprecated?

Reply:
test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go still calls e2enode.GetMasterAndWorkerNodes() and it takes more time for removing that.
It is a good idea to make it as deprecated.
I will update this PR soon.